esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -324,9 +327,13 @@ from database_settings import *
 MIDDLEWARE_LOCAL = []
 
 SECRET_KEY = '`openssl rand -base64 48`'
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+ALLOWED_HOSTS = ['$ESPHOSTNAME', '$SITENAME-orig.$DOMAIN']
```

Comment:
I think it'd be better to just stop creating -orig domains for new sites. Same applies for all uses below.